packages/badge/components/ColorCodedBadge.tsx

```diff
@@ -9,15 +9,16 @@ import { iconSizes } from "../../shared/styles/styleUtils/layout/iconSizes";
 interface ColorCodedBadgeProps {
   color?: React.CSSProperties["color"];
   iconShape?: SystemIcons;
+  children: JSX.Element | string;
 }
 
 const ICON_SIZE = "xxs";
 
-const ColorCodedBadge: React.FC<ColorCodedBadgeProps> = ({
+const ColorCodedBadge = ({
```

Comment:
I am not a TS expert, but I thought that the React.FC was helpful boilerplate for components. Was this something that we needed to change to support something else? Is it because children is not normally meant to be a string?

Reply:
This has been a recent discussion in the community about this and `React.FC` has been discouraged against. I have been trying to avoid it if possible. Due to how it implicitly takes children of type ReactNode, and issues with default props. [More Detail Here. ](https://github.com/facebook/create-react-app/pull/8177)

I have been following this handy [TypeScript Guide ](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/) which mentioned it as well.

Reply:
Okay great! way to lead us forward!